README.md

```diff
@@ -103,7 +112,7 @@ Requirements for parameters of training process:
 * <i>row_limit</i> - data type - integer
 * <i>drop_null</i> - data type - boolean, default value - False
 * <i>batch_size</i> - data type - integer, must be equal to or more than 1, default value - 32
-* <i>print_report</i> - data type - boolean, default value is False
+* <i>reports</i> - data type - if the value is passed through CLI - string, if the value is passed in the metadata file - string (if value is <i>"none"</i> or <i>"all"</i>) or list, accepted values: <i>"none"</i> (default) - no reports will be generated, <i>"all"</i> - generates both accuracy and sample reports, <i>"accuracy"</i> - generates an accuracy report, <i>"sample"</i> - generates a sample report, <i>"metrics_only"</i> - outputs the metrics information only to standard output. Default value is <i>none</i>. In the metadata file multiple values can be specified as a list to generate multiple types of reports simultaneously.
```

Comment:
May be add that value "metrics_only" doesn't generate reports in files?

Reply:
I will add this information.